### ISSUE TITLE
`resove` function of importmap has removed

### ISSUE THREAD

**Issue by TomokiMiyauci** [dalcib/esbuild-plugin-cache#3]
---
deno: `1.10.1`
esbuild-plugin-cache: `v0.2.6`
---

I got the following error:

```
Module '"https://deno.land/x/importmap@0.2.0/mod.ts"' has no exported member 'resolve'.
```

Apparently, starting with version `0.2.0` , the `resolve` function has been removed and the `resolveModuleSpecifier` has been changed to be exported directly.
https://deno.land/x/importmap@0.2.0/mod.ts#L239

**Comment by dalcib** [comment]
Thank you for inform.
I published a new version that fixed the importmap module at version 0.1.4.
It will allow the plugin to work properly. 
As soon as possible I will upgrade to the new importmap 0.2.0

**Comment by curiousercreative** [comment]
@dalcib this is resolved?

**Comment by dalcib** [comment]
Yes, it is resolved fixing the version of importmap in 0.1.4.
This also only affected the use of the plugin with Deno, not with Node.
